### ISSUE TITLE
DropdownButtonFormField semantics reads hint twice when no option is selected

### ISSUE THREAD

**Issue by kf6gpe** [flutter/flutter#52120]
https://b.corp.google.com/issues/150920084.

**Comment by HansMuller** [comment]
Version information (the output of Flutter doctor) and a simple test case are needed here.

**Comment by nscobie** [comment]
Hi all, I've been trying to come up with my own test case for this without context, but so far I've been unable to reproduce it. Do we know if this is still an issue? Thanks!

**My test case for reference:**
```dart
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}


// Trying to repro https://github.com/flutter/flutter/issues/52120
// Sample code originally from https://api.flutter.dev/flutter/material/DropdownButton-class.html

class MyApp extends StatelessWidget {
  static const String _title = 'Flutter Code Sample';

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: _title,
      home: Scaffold(
        appBar: AppBar(title: const Text(_title)),
        body: Center(
          child: MyStatefulWidget(),
        ),
      ),
    );
  }
}

class MyStatefulWidget extends StatefulWidget {
  MyStatefulWidget({Key key}) : super(key: key);

  @override
  _MyStatefulWidgetState createState() => _MyStatefulWidgetState();
}

class _MyStatefulWidgetState extends State<MyStatefulWidget> {
  String dropdownValue = null; // Note: I'm unsure how else to no select an option.

  @override
  Widget build(BuildContext context) {
    return DropdownButtonFormField<String>(
      hint: Tooltip(message: 'My Fancy Tooltip'),
      value: dropdownValue,
      onChanged: (String newValue) {
        setState(() {
          dropdownValue = newValue;
        });
      },
      items: <String>['One', 'Two', 'Three', 'Four']
          .map<DropdownMenuItem<String>>((String value) {
        return DropdownMenuItem<String>(
          value: value,
          child: Text(value),
        );
      }).toList(),
    );
  }
}
```

**Steps (using Pixel 3a):**
- Launch app, TalkBack says _"<app name>, flutter code sample, my fancy tooltip button, double tap to activate."_ (selection is blank since `dropdownValue = null`)
- Double tap, TalkBack says _"pop up menu, one, double tap to activate."_
- Exit pop up using back button, effectively keeping selection blank.
- TalkBack selects drop down and says _"flutter code sample, my fancy tooltip button, double tap to activate."_ (note hint is only said once).

**Output from `flutter doctor` for reference:**
```
[✓] Flutter (Channel master, 1.20.0-1.0.pre, on Mac OS X 10.15.5 19F101, locale en)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
[✓] Android Studio (version 4.0)
[✓] VS Code (version 1.46.0)
[✓] Connected device (2 available)
```

**Comment by tjonganthony** [comment]
Hey, you're right, I don't see it happening anymore! :)
Thanks for looking it up!

**Comment by nscobie** [comment]
Glad to hear it! Closing this out.
